### PR TITLE
Quickfix for stability improvements of interdomain tests

### PR DESCRIPTION
**Pull request by Mixaster995**
## Description
Fixes for some unstable points which made interdomain tests fail because of setup issues and not because of tests themselves.

### CODE REVIEW

File: examples/interdomain/README.md [hunk 1]
```diff
@@ -16,8 +16,8 @@ Interdomain tests can be on two clusters, for thus tests scheme of request will
 ## Requires
 
 - [Interdomain-DNS](./dns)
-- [Interdomain-spire](./spire)
 - [Load balancer](./loadbalancer)
```
Comment (denis-tingaikin) [inline examples/interdomain/README.md:19]: Let's make this first

File: examples/interdomain/usecases/Kernel2Vxlan2Kernel/README.md [hunk 2]
```diff
@@ -76,6 +76,11 @@ Deploy NSE:
...
 ```
 
+Wait for applications ready:
+```bash
+kubectl wait --for=condition=ready --timeout=5m pod -l app=nse-kernel -n ${NAMESPACE1}
+```
+
 Find NSE pod by labels:
 ```bash
 NSE=$(kubectl get pods -l app=nse-kernel -n ${NAMESPACE1} --template '{{range .items}}{{.metadata.name}}{{"\n"}}{{end}}')
```
Comment (denis-tingaikin) [inline examples/interdomain/usecases/Kernel2Vxlan2Kernel/README.md:86]:
```suggestion
NSE=$(kubectl get pods -l app=nse-kernel -n ${NAMESPACE1} --template '{{range .items}}{{.metadata.name}}{{"\n"}}{{end}}')
[[ ! -z $NSE ]]
```

File: examples/interdomain/usecases/Kernel2Vxlan2Kernel/README.md [hunk 3]
```diff
@@ -76,6 +76,11 @@ Deploy NSE:
 kubectl apply -k .
 ```
 
+Wait for applications ready:
+```bash
+kubectl wait --for=condition=ready --timeout=5m pod -l app=nse-kernel -n ${NAMESPACE1}
+```
+
```
Comment (denis-tingaikin) [inline examples/interdomain/usecases/Kernel2Vxlan2Kernel/README.md:83]:
```suggestion
```

```

File: examples/interdomain/usecases/FloatingKernel2Vxlan2Kernel/README.md [hunk 4]
```diff
@@ -88,6 +88,11 @@ Deploy NSE:
 kubectl apply -k .
 ```
 
+Wait for applications ready:
+```bash
+kubectl wait --for=condition=ready --timeout=5m pod -l app=nse-kernel -n ${NAMESPACE1}
+```
```
Comment (denis-tingaikin) [inline examples/interdomain/usecases/FloatingKernel2Vxlan2Kernel/README.md:94]:
```suggestion
```

File: examples/interdomain/README.md (hunk 1 above)
Comment (Mixaster995) [reply examples/interdomain/README.md:18]: ok, done

File: examples/interdomain/usecases/Kernel2Vxlan2Kernel/README.md (hunk 2 above)
Comment (Mixaster995) [reply examples/interdomain/usecases/Kernel2Vxlan2Kernel/README.md:81]: ok, done

File: examples/interdomain/usecases/FloatingKernel2Vxlan2Kernel/README.md (hunk 4 above)
Comment (Mixaster995) [reply examples/interdomain/usecases/FloatingKernel2Vxlan2Kernel/README.md]: ok, done

File: examples/interdomain/usecases/Kernel2Vxlan2Kernel/README.md (hunk 3 above)
Comment (Mixaster995) [reply examples/interdomain/usecases/Kernel2Vxlan2Kernel/README.md]: ok, done

File: examples/interdomain/usecases/FloatingKernel2Vxlan2Kernel/README.md [hunk 5]
```diff
@@ -93,6 +93,8 @@ Find NSE pod by labels:
 NSE=$(kubectl get pods -l app=nse-kernel -n ${NAMESPACE1} --template '{{range .items}}{{.metadata.name}}{{"\n"}}{{end}}')
 ```
 
+[[ ! -z $NSE ]]
```
Comment (denis-tingaikin) [inline examples/interdomain/usecases/FloatingKernel2Vxlan2Kernel/README.md:96]:
```suggestion
[[ ! -z $NSE ]]
```
```

File: examples/interdomain/usecases/Kernel2Vxlan2Kernel/README.md [hunk 6]
```diff
@@ -81,6 +81,8 @@ Find NSE pod by labels:
 NSE=$(kubectl get pods -l app=nse-kernel -n ${NAMESPACE1} --template '{{range .items}}{{.metadata.name}}{{"\n"}}{{end}}')
 ```
 
+[[ ! -z $NSE ]]
```
Comment (denis-tingaikin) [inline examples/interdomain/usecases/Kernel2Vxlan2Kernel/README.md:84]:
```suggestion
NSE=$(kubectl get pods -l app=nse-kernel -n ${NAMESPACE1} --template '{{range .items}}{{.metadata.name}}{{"\n"}}{{end}}')
[[ ! -z $NSE ]]
```

```

File: examples/interdomain/usecases/FloatingKernel2Vxlan2Kernel/README.md (hunk 5 above)
Comment (Mixaster995) [reply examples/interdomain/usecases/FloatingKernel2Vxlan2Kernel/README.md]: ok, done

File: examples/interdomain/usecases/Kernel2Vxlan2Kernel/README.md (hunk 6 above)
Comment (Mixaster995) [reply examples/interdomain/usecases/Kernel2Vxlan2Kernel/README.md]: ok, done

**Review by denis-tingaikin — APPROVED** [review]

**Pull request merged**